pages/1.12/release-notes/1.12.3/index.md

```diff
@@ -51,6 +51,8 @@ The issues that have been fixed in DC/OS 1.12.3 are grouped by feature, function
  
    Prior to this fix, the missing or empty file could prevent the agent from restarting and returning to normal operation. With this release, recovery from an empty or missing docker/volume file is handled by the containerizer or by the `docker/volume` isolator's `recover` method.
 
+- ASF-2731, COPS-4504 - Nvidia changed the container runtime settings in CUDA 10 images causing the GPU isolator in UCR to disable CUDA 10 images. Specifically, the new CUDA images relies on the `libnvidia-container` [library](https://github.com/NVIDIA/libnvidia-container) to set up the container runtime. This release updates the GPU isolator in UCR to workaround the changes required to support the image. 
+
```

Comment:
Add:


ASF-2719 - Agent could not recover due to empty docker volume checkpointed files.